### ISSUE TITLE
[Usability]A11y_AzureDataStudio_NewJobSteps_Keyboard:After hitting the "Delete step" button AI is not throwing any popup Whether the selected row is delated or not

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5824]
Check out [Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
GitHubTags: #A11yUsablity; #A11yTCS; #A11y_AzureDataStudio; #MacOs; #DesktopApp
**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS:MacOS Mojave 10.14.5
****Additional Details:***
*Tool Name: Keyboard
**Repro Steps:**
1.Launch the Azure Data Studio application.
2.Enable the SQL server agent extension.
3.Naviagte to the SQL agent tab from Home page->Click on new job button->Select steps tab.
4.Go to the "New step" button hit enter/spacebar 
5.navigate "to General" button and navigate with Tab till "Delete Step" button
6.Select the row in the general tab in table.
7.Hit enter to delete the row.
6.Observed there is no notification for the delated row.
**Actual:**
After hitting Delete step button there is no notification for the same whether the row is deleted or not .
**Expected:**
There should be notification for every actionable button so that user should know what action they have taken.
**User Impact:**
User will be confuse whether  the action is performed or not.
**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**Attachment for Reference:**
Voiceover is on while making an attachment.
[5824_AccessibilityA11y_AzureDataStudio_NewjobSteps_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3247659/5824_AccessibilityA11y_AzureDataStudio_NewjobSteps_ScreenReader.pptx)

**Does this issue occur when all extensions are disabled?: Yes**
